src/nrtk_explorer/app/export.py

```diff
@@ -0,0 +1,216 @@
+from pathlib import Path
+
+from trame.app import get_server, asynchronous
+from trame.widgets import quasar, html
+from trame.ui.quasar import QLayout
+
+from nrtk_explorer.app.applet import Applet
+from nrtk_explorer.library.dataset import (
+    discover_datasets,
+    dataset_select_options,
+)
+import nrtk_explorer.library.transforms as trans
+from nrtk_explorer.widgets.nrtk_explorer import ExportWidget
+
+
+def recursive_rmdir(path: Path):
+    if not path.is_dir():
+        return
+
+    for item in path.iterdir():
+        if item.is_file():
+            item.unlink()
+        else:
+            recursive_rmdir(item)
+
+    path.rmdir()
+
+
+class ExportApp(Applet):
+    def __init__(self, server):
+        super().__init__(server)
+
+        self.context.setdefault("repository", "")
+        self.state.setdefault("current_dataset", "")
```

Comment:
This is set in core.py too.  Should we refactor so we initialize the shared setup in once place?